docs/reference/search/explain.asciidoc

```diff
@@ -150,12 +141,10 @@ This will yield the same result as the previous request.
     string. Defaults to the default search analyzer.
 
 `analyze_wildcard`::
-    Should wildcard and prefix queries be analyzed or
-    not. Defaults to false.
+    Should wildcard and prefix queries be analyzed or not. Defaults to false.
```

Comment:
Another construct I'm not fond of "Shoud xyz or not." I'd probably go with, "Analyze wildcard and prefix queries."